.fauna.json

```diff
@@ -0,0 +1,4 @@
+{
+  "envPath": ".env.local",
+  "secretEnv": "FAUNADB_SECRET"
+}
```

Comment:
Do we checkin our secrets to git?

Reply:
No, `.env.local` is not tracked by git.

On local dev machines, this secret is unique per dev. README instructions were updated though I think this is how you have always set up local dev databases: https://github.com/twos-complement/level-protocol-website/pull/97/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R91

This would need to be configured for each vercel project though I suspect those envs are already established.

Reply:
OH this is just specifying where to look, I see now!